stowdir/.local/bin/csvdelta

```diff
@@ -181,6 +209,13 @@ def main(args):
 
         previous_value = value
 
+    output.close()
```

Comment:
Dang. You can't comment on a range that hasn't changed? That's... absurd.

Up a few lines, there's

```python
delta = None
if previous_value is not None:
    delta = value - previous_value
row += [delta]
```
which has the effect of inserting an empty cell in the CSV.

I think maybe the first row should be removed, or an explicit "N/A" placeholder used, or a default value inserted (if user chose).

I need to look closer at how `csvstats` consumes that empty cell.

Reply:
Empty cells are skipped

![image](https://github.com/Notgnoshi/dotfiles/assets/8878460/2af6f41c-b114-490e-979a-8d78b83af664)